src/Connection/RelayFactory.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+/*
+ * This file is part of the Predis package.
+ *
+ * (c) 2009-2020 Daniele Alessandri
+ * (c) 2021-2023 Till Krüss
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Predis\Connection;
+
+use InvalidArgumentException;
+use Predis\Command\RawCommand;
+use Predis\NotSupportedException;
+use Relay\Relay;
+
+class RelayFactory extends Factory
+{
+    /**
+     * @var string[]
+     */
+    protected $schemes = [
+        'tcp' => RelayConnection::class,
+        'unix' => RelayConnection::class,
+        'redis' => RelayConnection::class,
```

Comment:
This is missing TLS and `rediss`, right?

Reply:
Yeah, you're right. But they should be handled by `StreamConnection`, forgot to add it